src/frontend/screens/Settings/index.tsx

```diff
@@ -155,6 +159,9 @@ function Settings() {
               <Tabs.Panel value="accessibility">
                 <Accessibility />
               </Tabs.Panel>
+              <Tabs.Panel value="accounts">
+                <AccountSettings />
+              </Tabs.Panel>
```

Comment:
we should conditionally render these if achievements are enabled. again something we can do with a mobx class that default inits to false and runs the window.api check to update its value